### DISCUSSION TITLE
"Sort column: none" seems not working on REST API data sources

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#11543] [Help] [answered]
I'm testing my app on the cloud (v2.9.23) to see whether I can upgrade my self-host Budibase environment. While on it, I think I've found an unexpected behavior:

- I have a REST API data source without sort column. Naturally it honors the exact data row order that the server returns.
- While testing on the cloud, I found out that the data rows are forcefully sorted by the first column in the schema, even though the data source shows **Sort column: None** as below:
![image](https://github.com/Budibase/budibase/assets/93232582/1c39c4cd-10b2-43f9-9254-90bb0bfa1ba4)
- For comparison, I added a column with the same arbitrary value for every row, and set the **Sort column** to that one. Then the rows came back to the server's intended order (which is effectively an workaround for Sort column: None).

Based on the above, I think **Sort column: None** might be not working as expected. Although I was eager for the upgrade to use the new cool collaboration feature, but this unexpected behavior is not allowing it. Could you confirm this is a bug?

**Comment by andz-bb** [comment, marked as answer]
hey @dannyshin76, thanks for bringing this one to our attention. I have raised a ticket here for it: #11554

I think this should be addressed in the next Budibase release. In the meantime I would suggest that you continue to use the extra column in the response as the Sort column.

**Reply by dannyshin76** [reply]
Got that, I will check this problem again on the next release. Thanks!

**Comment by dannyshin76** [comment]
I have been regularly tracking this on the cloud and just found out this is finally fixed on v2.10.1. Good job!
